src/translations/en.json

```diff
@@ -2441,7 +2442,9 @@
             "edit_ui": "Edit in visual editor",
             "copy_to_clipboard": "Copy to clipboard",
             "triggers": {
-              "header": "Triggers",
+              "name": "Triggers",
+              "header": "When",
```

Comment:
Should we rename the key `header` as it was used before for other usage?

Reply:
We can remove them